G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/WordSave/ShouldSaveFile_WhenSaveCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,8 @@
+﻿♥filepath = ‴♥environment⟦TEMP⟧\tempwordfile.xlsx‴
```

Comment:
Check if the file exists, if it does, delete it.

Reply:
ping

Reply:
file.delete filename ♥filepath timeout 2